src/database/queries/userExtension.js

```diff
@@ -180,10 +180,12 @@ module.exports = class MenteeExtensionQueries {
 					stats: null,
 					tags: [],
 					configs: null,
-					visibility: null,
```

Comment:
@adithyadinesh0412 exclusion key change visibility -> menor_visibility
check if you can add the newly introduced keys here